lib/SIL/SILDeclRef.cpp

```diff
@@ -912,6 +912,8 @@ SubclassScope SILDeclRef::getSubclassScope() const {
   case AccessLevel::Public:
     return SubclassScope::Internal;
   case AccessLevel::Open:
+    if (classType->isResilient())
```

Comment:
`isResilient()` is the wrong check here; even a `@_fixed_layout` class will still be able to grow new methods in a resilient module.

Reply:
That's not how its implemented today. I guess you intend for @_fixed_layout classes to behave like their storage is exposed but methods and metadata is otherwise still resilient?